microprofile/grpc/metrics/src/main/java/io/helidon/microprofile/grpc/metrics/MetricsConfigurer.java

```diff
@@ -258,4 +166,17 @@ public String createName(ServiceDescriptor service, String methodName, MetricTyp
             return name;
         }
     }
+
+    private boolean isDiscovered(Method method) {
+        return GrpcMetricAnnotationDiscoveryObserverImpl.instance().isDiscovered(method);
```

Comment:
this method should be defined on the GrpcMetricAnnotationDiscoveryObserverImpl, and use static fields.